### PR TITLE
First page!

### LINKED ISSUE #4: Create basic input component
Create the bases for the the basic text input component that will be used through the application

### LINKED ISSUE #5: Create Home Screen

### PR DESCRIPTION
**Pull request by carlostrevisan1**
Built first page, first component, prettier and added a new font.

### CODE REVIEW

**Comment by carlostrevisan1** [comment]
image

File: src/components/StandardButton.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  button: {
+    width: 300,
+    backgroundColor: 'black',
+    borderRadius: 8,
+    paddingVertical: 12,
+    paddingHorizontal: 16,
+  },
+});
+
+export default StandardButton;
```
Comment (thalesgelinger) [inline src/components/StandardButton.tsx:42]: do we want 2 exports? couldn't we keep only one?

File: src/components/StandardTextInput.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    borderWidth: 2,
+    paddingTop: 12,
+    paddingRight: 48,
+    paddingBottom: 12,
+    paddingLeft: 14,
+  },
+  container: {},
+});
+
+export default StandardTextInput;
```
Comment (thalesgelinger) [inline src/components/StandardTextInput.tsx:39]: Same

**Review by thalesgelinger — COMMENTED** [review]
Take a look

**Pull request merged**